dts/arm/gigadevice/gd32f4xx/gd32f407Xe.dtsi

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
I found you give a upper case for the general character `X`, but we already have a `gd32f470ik.dtsi` file. And for gd32, general character `x` is lower case.
So, please consider to change the filename.

Reply:
Correct filenames.